### PR TITLE
"isort" Observed on Scripts

### PR DESCRIPTION
**Pull request by lnxpy**

### CODE REVIEW

**Comment by hirzel** [comment]
Hi, thanks for your pull request. Could you please provide some context? Is this just sorting import statements, or are there other changes as well? Also, note that it doesn't pass Travis yet, since MyPy finds 33 errors in schema2search_space.py.

**Comment by lnxpy** [comment]
Hi, it's my pleasure. Thanks.
I was looking at the scripts and I just noticed that import sections are a bit weak in production. Even though there is no problem with running the project, it's an optimization trick using [isort module](https://github.com/PyCQA/isort). `isort` only sorts the importing statements in the whole directory. I'm really confused about why Travis failed in the building process!!

Thank you for checking out.

**Comment by hirzel** [comment]
Seems like the change to the imports somehow changed the definition of `Schema`, since the first error is reported here: https://github.com/IBM/lale/blob/0b4dfcc422409215386fda41fe4a77375d5c8a15/lale/search/schema2search_space.py#L188

File: lale/search/schema2search_space.py [hunk 1]
```diff
@@ -12,24 +12,28 @@
...
+from typing import (Any, Dict, Iterable, Iterator, List, Optional, Set, Tuple,
+                    Union)
 
-from lale.util.Visitor import Visitor, accept
-from lale.util import VisitorPathError
-
-from typing import Any, Dict, List, Set, Iterable, Iterator, Optional, Tuple, Union
-from lale.schema_simplifier import findRelevantFields, narrowToGivenRelevantFields, simplify, filterForOptimizer
-
-from lale.schema_utils import Schema, getMinimum, getMaximum, STrue, SFalse, is_false_schema, is_true_schema, forOptimizer, has_operator, atomize_schema_enumerations, check_operators_schema
```
Comment (lnxpy) [inline lale/search/schema2search_space.py:27]: `isort` changed this line.. (line 27)

File: lale/search/schema2search_space.py [hunk 2]
```diff
@@ -12,24 +12,28 @@
...
-from lale.search.search_space import *
-from lale.search.lale_hyperopt import search_space_to_str_for_comparison
-from lale.search.PGO import PGO, FrequencyDistribution, Freqs
+import jsonschema
+import numpy
 
 from lale.operators import *
+from lale.schema_simplifier import (filterForOptimizer, findRelevantFields,
+                                    narrowToGivenRelevantFields, simplify)
+from lale.schema_utils import (Schema, SFalse, STrue,
```
Comment (lnxpy) [inline lale/search/schema2search_space.py:27]: .. to this. Not a big change! It's converted to a better style. All those modules are still imported.

**Review by lnxpy — COMMENTED** [review]
Here is what just happened. I've commented inline `schema2search_space.py` to show you the difference. Please checkout line 24. Thanks.

**Comment by hirzel** [comment]
Hi @lnxpy, can you fix the code in your pull request so the Travis build passes?

**Comment by lnxpy** [comment]
Hi, I can sort all import lists except the file that you mentioned. There might be a conflict with those methods in the `isord` module. I'll do that and create a brand-new PR. Thanks.

**Pull request closed without merging**